charts/simple-app/templates/deployment.yaml

```diff
@@ -337,9 +337,12 @@ spec:
 ---
 
 {{- if $.Values.podDisruptionBudget -}}
-{{- if lt ($.Values.replicaCount | int) 2 }}
+{{- if and $.Values.replicaCount (lt ($.Values.replicaCount | int) 2) }}
```

Comment:
Looks good - did you happen to test this to make sure it behaves as we expect?

Reply:
yea i tested it locally with the flask chart

Reply:
excellent... sorry i just merged a change right before yours... can you rebase, bump the version, and then let's get it in..